frontend/src/pages/exploreApplication/GetStartedPanel.tsx

```diff
@@ -36,16 +37,23 @@ type GetStartedPanelProps = {
 const GetStartedPanel: React.FC<GetStartedPanelProps> = ({ selectedApp, onClose, onEnable }) => {
   const { dashboardConfig } = useAppContext();
   const { enablement } = dashboardConfig.spec.dashboardConfig;
+  const [{ isInstalled, canInstall, error }, loaded] = useIntegratedAppStatus(selectedApp);
+
   if (!selectedApp) {
     return null;
   }
 
   const renderEnableButton = () => {
-    if (!selectedApp.spec.enable || selectedApp.spec.isEnabled) {
+    if (!selectedApp.spec.enable || selectedApp.spec.isEnabled || isInstalled) {
       return null;
     }
     const button = (
-      <Button variant={ButtonVariant.secondary} onClick={onEnable} isDisabled={!enablement}>
+      <Button
+        variant={ButtonVariant.secondary}
+        onClick={onEnable}
+        isDisabled={!enablement || !canInstall}
+        isLoading={!loaded && !error}
+      >
```

Comment:
As noted in our discussions there are some issues around this button; making them again here for posterity.

* Button doesn't hide after install (after modal closes)
* Button loads then hides if installed
    * I noticed this recently and didn't say anything -- but the `isLoading` probably should be a [Skeleton](https://www.patternfly.org/components/skeleton) component until if we know it should be there or not 
* Button shows disabled but doesn't show why -- before it was just `enablement` feature but now it's possible there is another feature flag or criteria from the API